build/root/WORKSPACE

```diff
@@ -37,9 +37,9 @@ versions.check(minimum_bazel_version = "0.23.0")
 
 http_archive(
     name = "io_k8s_repo_infra",
-    sha256 = "4a8384320fba401cbf21fef177aa113ed8fe35952ace98e00b796cac87ae7868",
-    strip_prefix = "repo-infra-df02ded38f9506e5bbcbf21702034b4fef815f2f",
-    urls = mirror("https://github.com/kubernetes/repo-infra/archive/df02ded38f9506e5bbcbf21702034b4fef815f2f.tar.gz"),
+    sha256 = "441631e653b281e2b71a2b1d1b69d7bcc827e08b81e7ae709134ae9f4c2afed9",
```

Comment:
grabbed with curl | sha256sum, hopefully correct!

Reply:
```suggestion
    sha256 = "4a8384320fba401cbf21fef177aa113ed8fe35952ace98e00b796cac87ae7868",
```

Reply:
nope, corrected it.